tests/SchemaTest.php

```diff
@@ -30,6 +30,18 @@ public function testGetIndexSchemas()
         }
     }
 
+    public function testGetPrimaryKeySchema() {
+        $indexes = $this->getConnection()->schema->getIndexSchemas();
+        foreach($indexes as $index) {
+            foreach($index->columns as $column) {
+                if ($column->isPrimaryKey) {
+                    $this->assertEquals($index->primaryKey, $column->name);
```

Comment:
Usually a test is added for the thing that is fixed. @klimov-paul is trying to tell that in this case it's not for the code that was changed. Is it true?